admin/src/assets/styles/common/_global.scss

```diff
@@ -181,6 +181,10 @@ body {
 		}
 	}
 
+	.w-100 {
```

Comment:
already existing `.limit` class few lines below

Reply:
Then, limit might not be the best name for this. But its okay, already removed it